Index: faq.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/faq.module	(revision )
+++ b/faq.module	(revision )
@@ -29,8 +29,8 @@
 /**
  * Implements hook_node_access().
  */
-function faq_node_access(\Drupal\node\NodeInterface $node, $op, $account, $langcode) {
+function faq_node_access(\Drupal\node\NodeInterface $node, $op, $account) {
   // Ignore non-FAQ node.
   if ($node->getType() !== 'faq') {
     return AccessResult::neutral();
   }
